src/Modules/User/Components/Login/LoginWrapper.tsx

```diff
@@ -46,12 +46,17 @@ export class LoginWrapperComponent extends BaseComponent<IProps, IState> {
     }
   }
 
+  public showNotificationLoginSuccess() {
+    this.showSuccess('Login successful!');
```

Comment:
i think we should declare a message file and then inject the message into this function.
btw, should use: login successfully. adv not adj